zebrad/tests/common/rpc_client.rs

```diff
@@ -44,4 +46,31 @@ impl RPCRequestClient {
     ) -> reqwest::Result<String> {
         self.call(method, params).await?.text().await
     }
+
+    /// Builds an RPC request, awaits its response, and attempts to deserialize
+    /// it to the expected result type.
+    ///
+    /// Returns Ok with json result from response if successful.
+    /// Returns an error if the call or result deserialization fail.
+    #[allow(dead_code)] // currently only used with getblocktemplate-rpcs feature
```

Comment:
Optional: we can say this in code so it's automatically checked:
```suggestion
    #[cfg(feature = "getblocktemplate-rpcs")]
```

Reply:
Fixed in [`516f17c`](https://github.com/ZcashFoundation/zebra/pull/5884/commits/516f17c257e3f39f63a9fa91ec0f0d253ca02b99).